assets/data-port/export/store/index.test.js

```diff
@@ -0,0 +1,197 @@
+import { select } from '@wordpress/data';
+import { apiFetch } from '@wordpress/data-controls';
+import registerExportStore, { EXPORT_STORE } from './index';
+import * as actions from './actions';
+
+jest.mock( '@wordpress/data-controls', () => {
+	const originalModule = jest.requireActual( '@wordpress/data-controls' );
+
+	return {
+		__esModule: true,
+		...originalModule,
+		apiFetch: jest.fn(),
+	};
+} );
+
+function mockActiveJob( job ) {
+	apiFetch.mockImplementation( () => {
+		if ( ! job )
+			throw {
+				code: 'sensei_data_port_job_not_found',
+				message: 'Not found',
+			};
+		else return job;
+	} );
+	select( EXPORT_STORE ).getJob();
+	apiFetch.mockClear();
+}
+
+describe( 'Export store', () => {
```

Comment:
I opted for higher level tests here, which test the store as a whole, checking that actions, resolvers, reducers work correctly together.